packages/react-flex/config/tests.js

```diff
@@ -2,6 +2,7 @@
 
 const { configure } = require('enzyme');
 const Adapter = require('enzyme-adapter-react-16');
+require('@testing-library/jest-dom');
```

Comment:
Maybe a comment why this import is required ? it's non standard across our packages

Reply:
Jest-DOM is an extension to Jest expecations. Allows me to simplify test styling with `.toHaveStyle`. What kind of comment would you expect to see and where should we have that?